modules/apps/callbacks/types/callbacks.go

```diff
@@ -87,11 +87,11 @@ func GetDestCallbackData(
 // address and gas limit from the callback data.
 func getCallbackData(
 	packetDataUnmarshaler porttypes.PacketDataUnmarshaler,
-	packet ibcexported.PacketI, remainingGas,
+	packetData []byte, srcPortID string, remainingGas,
 	maxGas uint64, callbackKey string,
 ) (CallbackData, error) {
 	// unmarshal packet data
-	unmarshaledData, err := packetDataUnmarshaler.UnmarshalPacketData(packet.GetData())
+	unmarshaledData, err := packetDataUnmarshaler.UnmarshalPacketData(packetData)
```

Comment:
just a nit, but maybe food for thought - wonder if it would read better if the arg was `data []byte` and rename `unmarshaledData` to `packetData`

Reply:
looks good to me, could also use `packetDataBz` instead of `data` if you find it better, but I don't mind